### PR TITLE
Solidus support v3.2

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,231 +1,44 @@
...
+RSpec/HookArgument:
   Enabled: false
 
-Metrics/CyclomaticComplexity:
+RSpec/MultipleExpectations:
   Enabled: false
 
-Metrics/LineLength:
+Rails/CreateTableWithTimestamps:
   Enabled: false
```
Comment (ccarruitero) [inline .rubocop.yml]: These additional configurations are necessary? I think we could update our code to use default solidus_dev_support rubocop

**Review by ccarruitero — COMMENTED** [review]
Seems good. just a minor comment

File: .rubocop.yml (hunk 1 above)
Comment (jtapia) [reply .rubocop.yml]: Changed it here https://github.com/magma-labs/solidus_payment_method_by_zone/pull/16/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR1

**Review by ccarruitero — APPROVED** [review]
If you can squash commits fixing rubocop into one commit would be great. Aside, looks good

**Comment by jtapia** [comment]
@ccarruitero uitero once is merge this one, will create the PR fixing the rubocop warnings, please let me know what do you think

**Comment by ccarruitero** [comment]
@jtapia Thanks for the updates. Sorry, I missed it. I'll merge it now

**Pull request merged**